aws-event-sources-operator/bundle/0.0.1/aws-event-sources-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,277 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "sources.triggermesh.com/v1alpha1",
+          "kind": "AwsEventSources",
+          "metadata": {
+            "name": "example-awseventsources"
+          },
+          "spec": {
+            "adapters": {
+              "awscodecommit": {
+                "image": "gcr.io/triggermesh/awscodecommitsource:v0.3.0"
+              },
+              "awscognitoidentity": {
+                "image": "gcr.io/triggermesh/awscognitoidentitysource:v0.3.0"
+              },
+              "awscognitouserpool": {
+                "image": "gcr.io/triggermesh/awscognitouserpoolsource:v0.3.0"
+              },
+              "awsdynamodb": {
+                "image": "gcr.io/triggermesh/awsdynamodbsource:v0.3.0"
+              },
+              "awskinesis": {
+                "image": "gcr.io/triggermesh/awskinesissource:v0.3.0"
+              },
+              "awssns": {
+                "image": "gcr.io/triggermesh/awssnssource:latest"
+              },
+              "awssqs": {
+                "image": "gcr.io/triggermesh/awssqssource:v0.3.0"
+              }
+            },
+            "affinity": {},
+            "fullnameOverride": "",
+            "image": {
+              "image": "gcr.io/triggermesh/aws-event-sources-controller:v0.3.0",
+              "pullPolicy": "IfNotPresent"
+            },
+            "imagePullSecrets": [],
+            "knative": {
+              "domain": "example.com",
+              "urlScheme": "http"
+            },
+            "nameOverride": "",
+            "nodeSelector": {},
+            "podAnnotations": {},
+            "podSecurityContext": {},
+            "rbac": {
+              "create": true
+            },
+            "resources": {
+              "requests": {
+                "cpu": "20m",
+                "memory": "20Mi"
+              }
+            },
+            "securityContext": {},
+            "serviceAccount": {
+              "annotations": {},
+              "create": true,
+              "name": ""
+            },
+            "tolerations": []
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Integration & Delivery,Streaming & Messaging
+    containerImage: gcr.io/triggermesh/aws-event-sources-operator:v0.0.1
+    createdAt: "2020-07-18T12:00:00Z"
```

Comment:
I think this shouldn't be here?

Reply:
It is metadata that displayed in the openshift operator hub UI. see https://redhat-connect.gitbook.io/certified-operator-guide/ocp-deployment/operator-metadata/creating-the-csv#fields-to-add-under-metadata-annotations

Reply:
Got it